### ISSUE TITLE
There is no training code for the new model.

### ISSUE THREAD

**Issue by kittinan** [tupleblog/generate-thai-lyrics#7]
There is no training code for the new model.

**Comment by titipata** [comment]
Yeah, I guess we can do major code refactoring. Removing all the old code basically.

**Comment by titipata** [comment]
I create PR here https://github.com/tupleblog/generate-thai-lyrics/pull/8. Can you add more instruction on how to run the web service in README @kittinan ?
